### ISSUE TITLE
Swallowed Error

### ISSUE THREAD

**Issue by johnpoth** [redhat-developer/service-binding-operator#909]
Currently, when referencing a service that does not exist in my service binding I get the following error:
```
{"level":"error","ts":1615816703.4740953,"logger":"reconciler","msg":"Failed to update SBR conditions","Request.Namespace":"camel-k","Request.Name":"sample","ServiceBinding.Name":"sample","sbr":{"apiVersion":"binding.operators.coreos.com/v1alpha1","kind":"ServiceBinding","namespace":"camel-k","name":"sample"},"error":"ServiceBinding.binding.operators.coreos.com \"sample\" is invalid: status.conditions.reason: Invalid value: \"\": status.conditions.reason in body should be at least 1 chars long"
```

which swallows the real error (the service doesn't exist).

**Comment by pedjak** [comment]
@johnpoth can you please check if this issue is solved in 0.7.0+?

**Comment by johnpoth** [comment]
Verified in 0.7.1, thanks !
